recipes/jsonapi-requests/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jsonapi-requests" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "d9438a15b752d74531edb48e5fa75d2387633b8daed2baacfbb3bec6231d2da2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest-runner
```

Comment:
Yes, you can see it in the original [setup.py line 15](https://github.com/socialwifi/jsonapi-requests/blob/98d408abae8047c566d4fbfba583c25524efa70f/setup.py#L15)